### PR TITLE
support x1 perception config

### PR DESCRIPTION
**Pull request by satoshi-ota**
## PR Type

- New Feature
- Improvement
- Bug Fix

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by yukkysaito** [comment]
@satoshi-ota 少し想定と違うところがあるので、定例でお話します！

**Comment by satoshi-ota** [comment]
@yukkysaito fix following points in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/75ae9f285c5e8c4fbea724d29c128d85cf254010, https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/b36b2371b7f7b9b28fa474ce965cad1a3003efdd

- remove config files depend on each products
- set xx1 perception config as default in ground_segmentation.param.yaml

**Comment by taichiH** [comment]
@satoshi-ota 
Why separate compare map components and ransac pipeline components?

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 1]
```diff
@@ -198,14 +289,74 @@ def launch_setup(context, *args, **kwargs):
...
+        parameters=[
+            {
+                "voxel_size_x": 0.4,
+                "voxel_size_y": 0.4,
+                "voxel_size_z": 100.0,
+                "voxel_points_threshold": 5,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
```
Comment (taichiH) [inline perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py:358]: It's not easy to read, so it's better to put these lines in a separete function.

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 2]
```diff
@@ -77,37 +79,133 @@ def create_additional_pipeline(vehicle_info, lidar_name):
...
+        remappings=[
+            ("input", "livox_concatenated/pointcloud"),
+            ("output", "short_height_obstacle_detection_area/pointcloud"),
+        ],
+        parameters=[
+            {
+                "input_frame": LaunchConfiguration("base_frame"),
+                "output_frame": LaunchConfiguration("base_frame"),
+                "min_x": 0.0,
+                "max_x": 19.6,  # max_x: 18.0m + base_link2livox_front_center distance 1.6m
```
Comment (yukkysaito) [inline perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py:127]: この辺のパラメータ追い出しくらいまではやりたいです

**Comment by satoshi-ota** [comment]
> @satoshi-ota Why separate compare map components and ransac pipeline components?

@taichiH There is no clear reason.

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 2 above)
Comment (satoshi-ota) [reply perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py]: fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/3a2a8516eed8985c103bb58792c47e25a0313efe

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 1 above)
Comment (satoshi-ota) [reply perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py]: fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/9ff854ce0da1d0f86d535a0c3563b08b3bdf995c

**Comment by satoshi-ota** [comment]
@yukkysaito please check modifications in 3a2a851, 9ff854c and approve :pray:

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 3]
```diff
@@ -257,7 +404,7 @@ def add_launch_arg(name: str, default_value=None):
 
     add_launch_arg("base_frame", "base_link")
     add_launch_arg("vehicle_param_file")
-    add_launch_arg("perception_config_file")
+    add_launch_arg("use_compare_map", "False")
```
Comment (yukkysaito) [inline perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py:407]: ground segmantation.yamlに入れていないのはなにか理由ありますか？

File: perception_launch/launch/perception.launch.xml [hunk 4]
```diff
@@ -38,9 +37,9 @@
         <include file="$(find-pkg-share perception_launch)/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py">
           <arg name="base_frame" value="base_link" />
           <arg name="vehicle_param_file" value="$(var vehicle_param_file)" />
+          <arg name="use_compare_map" value="false" />
```
Comment (yukkysaito) [inline perception_launch/launch/perception.launch.xml:40]: ground segmantation.yamlに入れていないのはなにか理由ありますか？

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 3 above)
Comment (satoshi-ota) [reply perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py:407]: There is no clear reason, but if it is better to include this param in yaml, I'll do it :+1:

File: perception_launch/launch/perception.launch.xml (hunk 4 above)
Comment (satoshi-ota) [reply perception_launch/launch/perception.launch.xml:40]: There is no clear reason, but if it is better to include this param in yaml, I'll do it :+1:

Comment (yukkysaito) [reply perception_launch/launch/perception.launch.xml:40]: Can you add it to a yaml file? :pray:

Comment (satoshi-ota) [reply perception_launch/launch/perception.launch.xml:40]: OK :+1: Thanks for your advise !

Comment (satoshi-ota) [reply perception_launch/launch/perception.launch.xml]: fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/f655b5254b750bec2f5baf98a00bd8101ded5c03

File: perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 3 above)
Comment (satoshi-ota) [reply perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py]: fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/129/commits/f655b5254b750bec2f5baf98a00bd8101ded5c03

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**